lib/improver/tests/psychrometric_calculations/psychrometric_calculations/test_FallingSnowLevel.py

```diff
@@ -103,90 +103,124 @@ def setUp(self):
+        """Test fills in missing data with orography + highest height"""
         plugin = FallingSnowLevel()
+        self.highest_wb_int[1, 1] = 100.0
         expected = np.array([[1.0, 1.0, 2.0],
-                             [1.0, 1.5, 2.0],
+                             [1.0, 301.0, 2.0],
                              [1.0, 2.0, 2.0]])
-        result = plugin.fill_in_missing_data(self.snow_level_data,
-                                             self.orog, self.land_sea,
-                                             self.highest_wb_int,
-                                             self.highest_height)
-        self.assertIsInstance(result, np.ndarray)
-        self.assertArrayEqual(result, expected)
-
-    def test_freezing_sealevel_point(self):
-        """Test sea point with integral below threshold sets snow level to 0"""
+        plugin.fill_in_high_snow_falling_levels(
+            self.snow_level_data, self.orog, self.highest_wb_int,
+            self.highest_height)
+        self.assertArrayEqual(self.snow_level_data, expected)
+
+    def test_no_fill_if_conditions_not_met(self):
+        """Test it doesn't fill in NaN if the heighest wet bulb integral value
+           is less the than theshold."""
         plugin = FallingSnowLevel()
-        orog = self.orog
-        orog[1, 1] = 0.0
-        land_sea = self.land_sea
-        land_sea[1, 1] = 0.0
         expected = np.array([[1.0, 1.0, 2.0],
-                             [1.0, 0.0, 2.0],
+                             [1.0, np.nan, 2.0],
                              [1.0, 2.0, 2.0]])
-        result = plugin.fill_in_missing_data(self.snow_level_data,
-                                             orog, land_sea,
-                                             self.highest_wb_int,
-                                             self.highest_height)
-        self.assertIsInstance(result, np.ndarray)
-        self.assertArrayEqual(result, expected)
+        plugin.fill_in_high_snow_falling_levels(
+            self.snow_level_data, self.orog, self.highest_wb_int,
+            self.highest_height)
+        self.assertArrayEqual(self.snow_level_data, expected)
+
+
+class Test_fill_sea_points(IrisTest):
 
-    def test_nonfreezing_sealevel_point(self):
-        """Test sea point with integral above threshold sets snow level
-        to highest_level"""
+    """Test the fill_in_missing_data method."""
+
+    def setUp(self):
+        """ Set up arrays for testing."""
+        self.snow_level_data = np.array([[1.0, 1.0, 2.0],
+                                         [1.0, np.nan, 2.0],
+                                         [1.0, 2.0, 2.0]])
+        self.wb_int = np.array([[100.0, 100.0, 100.0],
+                                [100.0, 5.0, 100.0],
+                                [100.0, 100.0, 100.0]])
+        self.land_sea = np.ones((3, 3))
+
+    def test_basic(self):
+        """Test it fills in the poitns it's meant to."""
```

Comment:
A small typo here, I think it should be "points."

Reply:
Fixed :)